package.json

```diff
@@ -77,16 +79,20 @@
     "power-assert": "1.4.2",
     "rimraf": "2.6.1",
     "semistandard": "10.0.0",
-    "swig": "1.4.2"
+    "source-map-support": "^0.4.14",
+    "swig": "1.4.2",
+    "typescript": "^2.2.2"
   },
   "scripts": {
-    "lint": "semistandard \"**/*.js\"",
-    "generate-apis": "node scripts/generate",
+    "lint": "semistandard \"samples/**/*.js\"",
+    "generate-apis": "npm run build-tools && node scripts/generate.js",
```

Comment:
Comically, no.  build-tools actually builds generate.ts into generate.js so it can be run by node.